deployment/terraform/remote_state.tf

```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "core" {
```

Comment:
I think this is good for now, there's a chance we'll have to go back and replace references to remote state with variables due to how this is going to be deployed in another AWS account. I think some of the resources we're able to gather information through remote state here will instead be configured manually or through another way and we'll be provided with IDs, etc. to use as variables.

I don't think anything needs to change in this PR since we don't have the information for that at the moment - however, in case it's easier to switch to using variables everywhere instead of remote state now rather than wait, I wanted to point that out.

Reply:
I think that based on the information we have now, we'll want to divorce ourselves from remote state in an error to be able to reuse the infrastructure configuration here across different types of environments without environment specific conditionals.